sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -72,10 +70,18 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
     this.mapboxMap =
       MapProvider.getMapboxMap(nativeMap, nativeObserver, mapInitOptions.mapOptions.pixelRatio)
     this.mapboxMap.renderHandler = renderer.renderThread.renderHandlerThread.handler
+
+    // telemetry
+    val telemetry: MapTelemetry = MapTelemetryImpl(
```

Comment:
As we reference the `MapTelemetryImpl` directly in the sdk, do we want to actually move the code inside the `module-telemetry` to the sdk itself? considering telemetry is not going to be a module after the change, keeping the module separate seems a bit confusing.